README.md

```diff
@@ -0,0 +1,5 @@
+# meteor-gazelle
```

Comment:
The old page may serve as inspiration. http://whatcd.github.io/Gazelle/

We should probably come up with a list of what should be on this page information wise...suggestions?


+1 otherwise